pkg/util/OWNERS

```diff
@@ -0,0 +1,10 @@
+reviewers:
+  - pmorie
```

Comment:
Generally we wouldn't add owners in the same PR.  Would be better to split this out.

Reply:
Actually @tallclair suggested for this. I am fine moving this out as well.